src/main/resources/lib/nextjs-proxy/parsing.es6

```diff
@@ -141,18 +141,24 @@ export const parseFrontendRequestPath = (req) => {
 
 
 export const relayUriParams = (req, frontendRequestPath) => {
-    const frontendServerUrl = getFrontendServerUrl();
-    const reqPath = frontendRequestPath?.length ? frontendRequestPath.replace(removeStartSlashPattern, '') : '';
+    let frontendServerUrl = getFrontendServerUrl();
+    let reqPath = frontendRequestPath?.length ? frontendRequestPath.replace(removeStartSlashPattern, '') : '';
```

Comment:
`const frontendServerUrl`
`const reqPath` 
?

Reply:
reqPath is reassigned later, but frontentServerUrl should be const